### PR TITLE
My String Calculator Kata

### PR DESCRIPTION
**Pull request by andresmoschini**
Hi team!

My implementation goes a little more beyond the kata specification, but I think it is consistent.

My tests differ from the [author base repository tests](https://github.com/wix/tdd-katas/blob/master/src/StringCalculator/StringCalculator.test.js) mainly because in my tests I paid special attention to the edge cases. 

I think that making honor and documenting the specifications with tests is essential, but also documenting the assumptions, for example, what happens with edge cases or unexpected scenarios.

In this journey, I have not only learned about TDD but also about converting _symbols_ to strings and, how _parseInt_ works.

### CODE REVIEW

File: src/andresmoschini/StringCalculator.test.ts [hunk 1]
```diff
@@ -14,12 +14,7 @@ describe("StringCalculator.add", () => {
     expect(result).toEqual(expectedResult);
   });
 
-  it.each([
-    { input: "   " },
-    { input: "abc" },
-    { input: ",5,6,7" },
-    { input: "x5x6x7x" },
-  ])(
+  it.each([{ input: "   " }, { input: "abc" }, { input: "x5x6x7x" }])(
```
Comment (darianbenito) [inline src/andresmoschini/StringCalculator.test.ts]: @andresmoschini I think that the code indentation changes are not in the commit scope (_string-calculator(step 3): return the sum of comma delimited values_)
```
it.each([
    { input: "   " },
    { input: "abc" },
    { input: "x5x6x7x" },
  ])(
```

Off-topic: I do not know why the suggestion button is hidden 

image

Comment (andresmoschini) [reply src/andresmoschini/StringCalculator.test.ts]: > @andresmoschini I think that the code indentation changes are not in the commit scope (string-calculator(step 3): return the sum of comma delimited values)

We are using prettier in all commits, so these kinds of changes are always included in the same commit in our javascript projects.

> Off-topic: I do not know why the suggestion button is hidden

Because it is already merged.

Comment (darianbenito) [reply src/andresmoschini/StringCalculator.test.ts]: > We are using prettier in all commits, so these kinds of changes are always included in the same commit in our javascript projects.

I understand! 👍 

> Because it is already merged.

Thanks! I think that GitHub is wrong here. What is the problem with the suggestion option in a merged PR?

Comment (andresmoschini) [reply src/andresmoschini/StringCalculator.test.ts]: I do not know, but I suppose that a suggestion has no sense when the PR was already merged.

image

You can write it manually anyway:

```suggestion
test
```

Comment (darianbenito) [reply src/andresmoschini/StringCalculator.test.ts]: > You can write it manually anyway:

_hecha la ley hecha la trampa_
